src/pages/index.js

```diff
@@ -59,25 +145,83 @@ const newCardPopup = new PopupWithForm(".popup-card", (formValues) => {
     link: formValues["second-input"],
     alt: formValues["first-input"],
   };
-  const newCard = createCard(cardData);
-
-  section.addItem(newCard, "prepend");
-  newCardPopup.close();
-  addFormValidator.resetForm();
-  addFormValidator.disableSubmitButton();
+  const cardPopupButton = newCardPopup._submitButton;
```

Comment:
`_submitButton`

* Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.